scene/3d/light_3d.cpp

```diff
@@ -276,6 +276,11 @@ void Light3D::_update_visibility() {
 
 void Light3D::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_TRANSFORM_CHANGED: {
+			if (Engine::get_singleton()->is_editor_hint()) {
```

Comment:
The condition is not needed, because `update_configuration_warnings()` is no-op in exported projects. Other usages of this method don't do any checks.

Reply:
Ok then! While at it, I went over the codebase with `if \(Engine::get_singleton\(\)->is_editor_hint\(\)\) {\n\t*update_c` and purged the codebase from these.

Man how I love negative diffs!